docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites.md

```diff
@@ -76,9 +82,24 @@ Installing YugabyteDB Anywhere on Airgapped hosts, without access to any Interne
 
 ### Kubernetes-based installations
 
-For a [Kubernetes-based installation](#kubernetes-based-installations), you need to ensure that the host can pull container images from the [Quay.io](https://quay.io/) container registry. For details, see [Pull and push YugabyteDB Docker images to private container registry](#pull-and-push-yugabytedb-docker-images-to-private-container-registry). 
+The YugabyteDB Anywhere Helm chart has been tested using the following software versions:
+
+- Kubernetes 1.22
+- Helm 3.10
 
-In addition, you need to ensure that core dumps are enabled and configured on the underlying Kubernetes node. For details, see [Specify ulimit and remember the location of core dumps](#specify-ulimit-and-remember-the-location-of-core-dumps). 
+
+Before installing the YugabyteDB Admin Console, verify that you have the following:
```

Comment:
What is the YugabyteDB Admin Console?

Reply:
Ah, good catch, it's one of the legacy names for YBA